tests/slow/src/test/scala/tests/feature/FileDecoderProviderLspSuite.scala

```diff
@@ -0,0 +1,683 @@
+        |    64:           SHAREDtype 55
+        |    66:         APPLY(13)
+        |    68:           SELECTin(8) 29 [<init>[Signed Signature(List(java.lang.Class),scala.runtime.ModuleSerializationProxy) @<init>]]
+        |    71:             NEW
+        |    72:               TYPEREF 25 [ModuleSerializationProxy]
+        |    74:                 TERMREFpkg 24 [scala[Qualified . runtime]]
+        |    76:             SHAREDtype 72
+        |    78:           CLASSconst
+        |    79:             SHAREDtype 45
+        |    81:         PRIVATE
+        |    82:         SYNTHETIC
+        |    83:       DEFDEF(9) 1 [foo]
+        |    86:         EMPTYCLAUSE
+        |    87:         IDENTtpt 19 [Unit]
+        |    89:           TYPEREF 19 [Unit]
+        |    91:             TERMREFpkg 20 [scala]
+        |    93:         UNITconst
+        |    94:     OBJECT
+        |    95:     SYNTHETIC
+        |    96:     ANNOTATION(16)
+        |    98:       TYPEREF 30 [SourceFile]
+        |   100:         TERMREFpkg 34 [scala[Qualified . annotation][Qualified . internal]]
+        |   102:       APPLY(10)
+        |   104:         SELECTin(6) 38 [<init>[Signed Signature(List(java.lang.String),scala.annotation.internal.SourceFile) @<init>]]
+        |   107:           NEW
+        |   108:             SHAREDtype 98
+        |   110:           SHAREDtype 98
+        |   112:         STRINGconst 39 [app/src/main/scala/Main.scala]
+        |   114:
+        |
+        | 43 position bytes:
+        |   lines: 4
+        |   line sizes: 23, 11, 20, 0
+        |   positions:
+        |         0: 0 .. 56
+        |         4: 36 .. 36
+        |         7: 36 .. 36
+        |        25: 36 .. 56
+        |        28: 36 .. 56
+        |        36: 36 .. 36
+        |        45: 36 .. 36
+        |        49: 36 .. 36
+        |        53: 36 .. 36
+        |        58: 36 .. 36
+        |        62: 36 .. 36
+        |        72: 36 .. 36
+        |        78: 36 .. 36
+        |        83: 36 .. 56
+        |        87: 47 .. 51
+        |        93: 54 .. 56
+        |
+        | source paths:
+        |         0: app/src/main/scala/Main.scala
+        |
+        |
+        | 0 comment bytes:
+        |""".stripMargin
+
+  private val javap =
+    s"""|Compiled from "Main.scala"
+        |public class foo.bar.example.Foo {
+        |  public foo.bar.example.Foo();
+        |}
+        |""".stripMargin
+
+  private val javapToplevel =
+    s"""|Compiled from "Main.scala"
+        |public final class foo.bar.example.Main$$package {
+        |  public static void foo();
+        |}
+        |""".stripMargin
+
+  private[this] val format = new SimpleDateFormat("d MMM yyy;")
+  private def javapVerbose(workspace: String) =
+    s"""|Classfile $workspace/foo/bar/example/Foo.class
+        |  Last modified ${format.format(new Date())} size 312 bytes
+        |  SHA-256 checksum fae10a1f13dd3fbccf35abc0dec9ec5189b302782a75868a5afed87f6abb1926
```

Comment:
Date format and algorithm used for hashing differ among JDK's :/ Perhaps it should be stripped in tests